actions/promote_package.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: promote_package
+runner_type: mistral-v2
+description: Promote a package to production from staging.
+enabled: true
+entry_point: workflows/promote_package.yaml
+parameters:
+  package:
+    type: string
+    description: Package to promote
+    required: true
+  version:
+    type: string
+    description: Version to promote
+    required: true
+  revision:
+    type: string
+    description: Revision to promote
+    required: true
+  os:
+    description: OS to promote (will promote all if not specified)
+    required: true
+    default: all
```

Comment:
Ah, see: https://circle-artifacts.com/gh/StackStorm/st2-packages/607/artifacts/0/home/ubuntu/packages/payload.json
It's already unsync.

It's metadata for latest build.
This file states which `versions`, `revisions` build just generated.